### DISCUSSION TITLE
RT questions

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio#3138] [General]
I saw just the post [Real-time multi-user collaborative diagramming](https://drawio-app.com/real-time-multi-user-collaborative-diagramming/). Does this mean that realtime collaboration is likely to come to DrawIO/DiagramsNet (both hosted and self-hosted) as well? Would be a game-changer for us as we are looking for alternative whiteboard solutions but multi-collab DrawIO would probably cover most of them.

_Originally posted by @lfvjimisola in https://github.com/jgraph/drawio/issues/1396#issuecomment-1294898468_

**Comment by lfvjimisola** [comment]
@davidjgraph What's your involvement via DrawIO? 

You repost above twisted the text a bit. Reposting here:


> [Real-time multi-user collaborative diagramming](https://drawio-app.com/real-time-multi-user-collaborative-diagramming/). Does this mean that realtime collaboration is likely to come to DrawIO/DiagramsNet (both hosted and self-hosted) as well? Would be a game-changer for us as we are looking for alternative whiteboard solutions but multi-collab DrawIO would probably cover most of them.

**Comment by davidjgraph** [comment]
That's not actually our post, just to confuse things. drawio-app.com is the Atlassian integration vendor, they have nothing to do with this project.

It only relates to Confluence Cloud. That said, RT is present in the core project. diagrams.net has RT if you select a storage that supports it, like Google Drive or OneDrive.

The self-hosted solution doesn't have any storage, without storage there can't be RT collab.

**Comment by NamUKF** [comment]
Hi, 

the self-hosted diagrams.net, does support a nextcloud integration https://github.com/jgraph/drawio-nextcloud. it is mentioned, that RT collab is only support when using the official web version of diagrams.net. Is there any intention, that it'll work for the selfhosted version ?

**Comment by biwhite** [comment]
I'm using self hosted with Google Drive backend.  Would real-time collaboration work for me?  I've attempted with the AWS IOT bits, but not getting anywhere.
